_posts/2020-11-06-JOIN-2020.md

```diff
@@ -0,0 +1,330 @@
+    <iframe src="https://www.youtube.com/embed/PayZ4lKsr-w" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+I don't like failing.
+I want to achieve a goal, I invest my time and energy, I am looking forward to achieving this goal, I am getting closer, closer - and fail.
+The goal is not achieved and I want to retreat into a dark corner and just forget.
+This is a perfectly normal human reaction.
+Yet, as we all know, no big endeavor happens without its fair share of failure - some huge achievements even happen out of failure alone...
+Hence, there is something useful to it.
+Something that even smells like success!
+This talk analyses different projects, their main impediments and success or failure factors, includes some interactive parts - and in the end (hopefully) leaves us with some renewed courage to face the failures in our lives... to gain value from them.  
+
+### Session 4 - Why don't you get my code? - Francis Laleman
+
+**Video under review - will be added soon!**
+
+This is a playful, interactive session on the concept of "code" - as a formal, grammar-and-syntax-driven substitute for "meaning".
+We all know that code is the central edifice of what it is that we are doing - at work, and, perhaps, even in life.
+But do we fully understand the profound relationship between code and meaning? Do we really grasp how code, in all its formality and conventions, both pins us down into the confinement of "knowns", and has the power to liberate us from what is already known and drive us forward?
+After this session, expect to be aware of more than just "code", and to be able to understand "meaning", in life and work, through a variety of grammars and syntaxes, more diverse than ever imagined.  
+
+## In the spotlight - Ghelamco alert
+
+This talk features the awesome Internship project by Kevin Govearts which is a shining example of and end-to-end integrated IoT project.
+It touches a lot of interesting technologies and integrates everything very nicely and includes managing a Raspberry Pi device via AWS.
+
+The project provides a solution to our parking problem at our Ghelamco office in Ghent.
+Since the parking lot needs to be vacated whenever there is a "home" soccer match or we get a hefty fine per car that remains, we needed an automated solution to warn the people at the office.
+This project does just that, end-to-end.
+It reads & scrapes the data and determines when the matches are and commands the Raspberry Pi to light up the alarm light.
+
+It uses at least 10 different AWS services and the usage of AWS Greengrass allows you to deploy production ready applications to edge devices without having to manage it yourself.
+I wont go into all the technical details since you should absolutely read Bas and Kevin's their [excellent blog post](https://ordina-jworks.github.io/cloud/2020/09/28/ghelamco-alert.html){:target="_blank" rel="noopener noreferrer"} about the project!
```

Comment:
```suggestion
I won't go into all the technical details since you should absolutely read Bas and Kevin's [excellent blog post](https://ordina-jworks.github.io/cloud/2020/09/28/ghelamco-alert.html){:target="_blank" rel="noopener noreferrer"} about the project!
```